swagger.json

```diff
@@ -40,9 +40,160 @@
+                "operationId": "create_user",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    { "$ref": "#/parameters/createUserParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/User"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/users/{user_id}": {
+            "get": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users/{user_id:[0-9]+}",
+                "description": "Get a user.",
+                "operationId": "get_user",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "user_id",
+                        "in": "path",
+                        "required": true,
+                        "description": "ID of the user.",
+                        "type": "integer",
+                        "format": "int64"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User record.",
+                        "schema": {
+                            "$ref": "#/definitions/User"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users/{user_id:[0-9]+}",
+                "description": "Remove a user from your organization.",
+                "operationId": "delete_user",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "user_id",
+                        "in": "path",
+                        "required": true,
+                        "description": "ID of the user.",
+                        "type": "integer",
+                        "format": "int64"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The user was successfully removed."
```

Comment:
Long term I'd like to start returning `204: No Content` for successful DELETE operations that don't return any response.

Reply:
Do you want that change made here?

Reply:
Your call :)

Reply:
Deferred this.